DotnetFoundation/DotnetFoundation.Api/Controllers/TaskDetailsController.cs

```diff
@@ -0,0 +1,147 @@
+using DotnetFoundation.Application.Interfaces.Services;
+using DotnetFoundation.Application.Models.Common;
+using DotnetFoundation.Application.Models.DTOs.TaskDetailsDTO;
+using DotnetFoundation.Application.Models.Enums;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc;
+
+namespace DotnetFoundation.Api.Controllers;
+
+[ApiController]
+[Route("api/tasks")]
+public class TaskDetailsController : ControllerBase
```

Comment:
The controller name should be just "TaskController".
The DBO Model name should be "Task".
The table name should be "tasks"

Reply:
We need not worry about the consistency in between the database table name and the entity. As per the SQL naming standards, the table name should be plural and should follow snake casing.

Reply:
Renamed table and controller